pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -246,6 +257,94 @@ func GatherMostRecentMetrics(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+// GatherContainerImages collects essential information about running containers.
+// Specifically, the age of pods, the set of running images and the container names are collected.
+//
+// Location in archive: config/running_containers.json
+func GatherContainerImages(i *Gatherer) func() ([]record.Record, []error) {
+	return func() ([]record.Record, []error) {
+		records := []record.Record{}
+
+		// Cache for the temporary image count list.
+		img2month2count := img2Month2CountMap{}
+
+		// Use the Limit and Continue fields to request the pod information in chunks.
+		continueValue := ""
+		for {
+			pods, err := i.coreClient.Pods("").List(i.ctx, metav1.ListOptions{
+				Limit:    imageGatherPodLimit,
+				Continue: continueValue,
+				// FieldSelector: "status.phase=Running",
```

Comment:
Just a nitpick. We can remove this line right? I noticed we have it in master as well...

Reply:
@tremes We could remove it if you want, but at this point I'd rather keep it there for consistency. I believe I left it there intentionally to indicate that it was unnecessary or something like that.

Reply:
Yeah...let's keep there.